lib/Net/Twitter/Role/API/Media.pm

```diff
@@ -0,0 +1,103 @@
+package Net::Twitter::Role::API::Media;
+
+use Moose::Role;
+use Net::Twitter::API;
+use DateTime::Format::Strptime;
+use namespace::autoclean;
+
+
+has media_api_url => (
+    isa => 'Str',
+    is => 'rw',
+    default => 'http://api.twitter.com/1.1',
+);
```

Comment:
Why specify this vs just using the one specified in `Net::Twitter::Role::API::RESTv1_1`?
Could you just call
`with "Net::Twitter::Role::API::RESTv1_1";` instead?